### PR TITLE
Print version of created schema

### PR DESCRIPTION
**Pull request by busybeetree**
https://hivemq.kanbanize.com/ctrl_board/4/cards/14849/details/

### CODE REVIEW

File: src/main/java/com/hivemq/cli/hivemq/schemas/CreateSchemaTask.java [hunk 1]
```diff
@@ -70,13 +75,24 @@ public boolean execute() {
...
 
+        if (printEntireSchema) {
+            outputFormatter.printJson(createdSchema);
+        } else {
+            final JsonObject versionObject = new JsonObject();
+            if (createdSchema.getVersion() != null) {
+                versionObject.add(Schema.SERIALIZED_NAME_VERSION, new JsonPrimitive(createdSchema.getVersion()));
+            }
+            outputFormatter.printJson(versionObject);
+        }
```
Comment (gitseti) [inline src/main/java/com/hivemq/cli/hivemq/schemas/CreateSchemaTask.java:94]: I am not sure if we should print the JSON response from the server for solely one create command. 
It seems a bit inconsistent with the policy create and the other delete commands.
I see three ways of how we could handle the ouput of REST API commands:
1. Never print the JSON response. Users would need to get the version via the get command. (I think this is the option chosen most widely)
2. Never print the JSON response and offer an opt-in option, as you did with `--print-result`, which can be combined with `--fields` to only get the version.
3. Always print the JSON response. (Stripe does this for example: https://stripe.com/docs/cli/delete)

File: src/main/java/com/hivemq/cli/hivemq/schemas/CreateSchemaTask.java [hunk 2]
```diff
@@ -70,13 +75,22 @@ public boolean execute() {
...
             outputFormatter.printApiException("Failed to create schema", apiException);
             return false;
         }
 
+        if (printVersion) {
+            final JsonObject versionObject = new JsonObject();
+            if (createdSchema.getVersion() != null) {
+                versionObject.add(Schema.SERIALIZED_NAME_VERSION, new JsonPrimitive(createdSchema.getVersion()));
+            }
+            outputFormatter.printJson(versionObject);
```
Comment (gitseti) [inline src/main/java/com/hivemq/cli/hivemq/schemas/CreateSchemaTask.java:91]: Does it need to be wrapped in JSON?
We could just output the version like 
```
mqtt hivemq create schema ...
1
```
By doing this, the output is directly usable through scripting without having to mingle with the json first.

Comment (busybeetree) [reply src/main/java/com/hivemq/cli/hivemq/schemas/CreateSchemaTask.java:91]: Errors are already printed in JSON, e.g. `{"errors":[{"title":"Schema already present","detail":"The given schema is already present as the latest version for the schema id 'schema-1'."}]}`

I think JSON keeps this consistent, and allows for the possibility of other printed fields in future.

Comment (gitseti) [reply src/main/java/com/hivemq/cli/hivemq/schemas/CreateSchemaTask.java:91]: Makes sense. Then let's keep it as it is. ✅

**Pull request merged**